api/tests/test_smoke.py

```diff
@@ -186,6 +186,12 @@ def generate_end_user_header(env):
     return "RJ11"
 
 
+def generate_end_user_receiver_header(env):
+    if env == "prod":
+        return "XXXX"
+    return "DEF"
```

Comment:
Whats the idea behind this? it will always be RJ11.DEF for other environments biut would be XXXX.XXXX for prod?

Reply:
Yeah, similar to how we're generating the `NHSD-End-User-Organisation-ODS`